src/Providers/ReviewsIo/ServiceActions.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\ReviewsIo;
+
+use PodPoint\Reviews\ActionsInterface;
+use PodPoint\Reviews\ApiClientInterface;
+use PodPoint\Reviews\Exceptions\ValidationException;
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\GetServiceReviews;
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\EmailInviteRequest;
+use PodPoint\Reviews\Providers\ReviewsIo\Request\Service\FindReviewRequest;
+
+/**
+ * Class ServiceActions
+ * @package PodPoint\Reviews\Providers\ReviewsIo
+ */
+class ServiceActions implements ActionsInterface
+{
+    /**
+     * @var ApiClientInterface
+     */
+    protected $apiClient;
+
+    /**
+     * @var string
+     */
+    protected $store;
+
+    /**
+     * ServiceReview constructor.
+     *
+     * @param $apiClient
+     */
+    public function __construct(ApiClientInterface $apiClient)
+    {
+        $this->apiClient = $apiClient;
+    }
+
+    /**
+     * @param array $options
+     *
+     * @return array|mixed
+     * @throws ValidationException
+     */
+    public function invite(array $options)
+    {
+        $options['store'] = $this->store;
+
+        $request = new EmailInviteRequest($this->apiClient, $options);
+
+        return $request->send();
+    }
+
+    /**
+     * @param array $options
+     *
+     * @return array|mixed
+     * @throws ValidationException
+     */
+    public function getReviews(array $options)
```

Comment:
I think here should be

```
public  function getReviews(array $options = [])
```

so we can `...->getReviews()` instead of having to do`...->getReviews([])` to prevent error